src/screens/OrganizationPeople/OrganizationPeople.spec.tsx

```
@@ -784,6 +853,264 @@ describe('Organization People Page', () => {
+        variables: {
+          userid: 'test-user-id',
+          orgid: linkURL,
+        },
+      },
+      result: mockCreateMemberResponse2,
+    },
+  ];
+
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)